amplipi/streams.py

```diff
@@ -25,14 +25,16 @@
 import sys
 import subprocess
 import time
-from typing import Union
+from ctypes import c_ulong # unsigned ints for MAC generation
+from typing import Union, Optional
 import threading
 
 # Used by InternetRadio and Spotify
 import json
 import signal
 import ast
 import socket
+import hashlib # md5 for string -> MAC generation
```

Comment:
do you still need to import this?

Reply:
yes, its still used to generate the MAC from string. Hashlib appears to be part of the standard library tho.